doc/dnf5.conf.5.rst

```diff
@@ -0,0 +1,1366 @@
+
+    Default: ``False``.
+
+.. _installonlypkgs_options-label:
+
+``installonlypkgs``
+    :ref:`list <list-label>`
+
+    List of provide names of packages that should only ever be installed, never
+    upgraded. Kernels in particular fall into this category.
+    These packages are never removed by ``dnf autoremove`` even if they were
+    installed as dependencies (see
+    :ref:`clean_requirements_on_remove <clean_requirements_on_remove_options-label>`
+    for auto removal details).
+    This option append the list values to the default installonlypkgs list used
+    by DNF5. The number of kept package versions is regulated
+    by :ref:`installonly_limit <installonly_limit_options-label>`.
+
+.. _installonly_limit_options-label:
+
+``installonly_limit``
+    :ref:`integer <integer-label>`
+
+    Number of :ref:`installonly packages <installonlypkgs_options-label>` allowed to be installed
+    concurrently.
+
+    ``1`` is explicitely not allowed since it complicates kernel upgrades due to protection of
+    the running kernel from removal.
+
+    Minimum is ``2``.
+
+    ``0`` means unlimited number of installonly packages.
+
+    Default: ``3``.
+
+.. _installroot_options-label:
+
+``installroot``
+    :ref:`string <string-label>`
+
+    The root of the filesystem for all packaging operations.
+    It requires an absolute path.
+    See also :ref:`--installroot commandline option <installroot_options-label>`.
+
+    Default: ``/``.
+
+.. _install_weak_deps_options-label:
+
+``install_weak_deps``
+    :ref:`boolean <boolean-label>`
+
+    If enabled, when a new package is about to be installed, all packages linked by weak dependency
+    relation (Recommends or Supplements flags) with this package will be pulled into the transaction.
+
+    Default: ``True``.
+
+.. _keepcache_options-label:
+
+``keepcache``
+    :ref:`boolean <boolean-label>`
+
+    If enabled, keeps downloaded packages in the cache. If disabled cache will persist
+    until the next successful transaction even if no packages have been installed.
+
+    Default: ``False``.
+
+.. _logdir_options-label:
+
+``logdir``
+    :ref:`string <string-label>`
+
+    Directory where the log files will be stored.
+
+    Default: ``/var/log``.
+
+.. _log_rotate_options-label:
+
+``log_rotate``
+    :ref:`integer <integer-label>`
+
+    Log files are rotated ``log_rotate`` times before being removed.
+    If ``log_rotate`` is ``0``, the rotation is not performed.
+
+    Default: ``4``.
+
+.. _log_size_options-label:
+
+``log_size``
+    :ref:`storage size <storage_size-label>`
+
+    Log  files are rotated when they grow bigger than ``log_size`` bytes. If
+    ``log_size`` is ``0``, the rotation is not performed.
+
+    The size applies for individual log files, not the sum of all log files.
+    See also :ref:`log_rotate <log_rotate_options-label>`.
+
+    Default: ``1M``.
+
+.. _metadata_timer_sync_options-label:
+
+``metadata_timer_sync``
```

Comment:
I thought `makecache timer` is not supported anymore? If so, we should drop this.